### ISSUE TITLE
'tcprewrite' with fragroute option fails on oversized packets

### ISSUE THREAD

**Issue by recvfrom** [appneta/tcpreplay#466]
'tcprewrite' with the fragroute option can't handle fragmenting packets collected from an interface where the MTU size is greater than 1500 bytes.  This issue was actually asked about on the mailing list a while back, and a solution proposed - the thread can be found at:

https://sourceforge.net/p/tcpreplay/mailman/message/32419069/

Could such a fix be implemented in the main codebase so that users don't have to modify and rebuild source to get this functionality?  I can think of two instances where this might be useful:
1. 'tcprewrite' of a pcap file collected on an interface that supports jumbo frames (9k MTU)
2. 'tcprewrite' of a pcap file collected from the loopback interface (65K MTU)

Maybe this could be implemented as a command line option taking three arguments (accepting values 'standard', 'jumbo', and 'max') and behind the scenes it could choose an underlying struct with enough space in the pkt_buf?  Alternatively, the exact MTU size to use could be specified at the command line, although this might have performance implications depending on how these structures are allocated at runtime.  What do you think?  I'd be happy to take a first shot at implementing and submit a PR if you'd like.  Thanks!

**Comment by fklassen** [comment]
Will investigate and consider for 4.3

**Comment by fklassen** [comment]
fixed in PR #509 

To test, create `frag.cfg` that contains `ip_frag 1500`. Then run:

```
tcprewrite --fragroute=frag.cfg --infile=9000.pcap --outfile=1500.pcap
```

[9000.pcap.zip](https://github.com/appneta/tcpreplay/files/2511772/9000.pcap.zip)

**Comment by fklassen** [comment]
Reopen to fix potential segfault

**Comment by fklassen** [comment]
fix latest bug in PR #516
